gtk-3.yaml

```
@@ -1,14 +1,17 @@
 package:
   name: gtk-3
   version: "3.24.51"
-  epoch: 0
+  epoch: 1
   description: The GTK+ Toolkit (v3)
   copyright:
     - license: LGPL-2.1-or-later
   scriptlets:
     post-install: |
       #!/bin/sh
       glib-compile-schemas /usr/share/glib-2.0/schemas
+  dependencies:
+    runtime:
+      - gtk-update-icon-cache
```

```suggestion
      - ${{package.name}}-update-icon-cache
```

https://github.com/wolfi-dev/os/pull/71974/files#r2524721803